src/Shared/Recruit.Vacancies.Client/Infrastructure/QueryStore/Projections/ApplicationReview/ApplicationReviewSummary.cs

```diff
@@ -6,5 +6,6 @@ public class ApplicationReviewSummary
     {
         public long VacancyReference { get; set; }
         public ApplicationReviewStatus Status { get; set; }
+        public bool IsWithdrawn { get; set; }
```

Comment:
This can be a calculated field if we use status to identify the application is withdrawn

Reply:
See above comment